src/daemon/daemon.cpp

```diff
@@ -2435,7 +2435,7 @@ try
                 grpc::Status{grpc::INVALID_ARGUMENT, "Multipass can only take snapshots of stopped instances."});
 
         auto snapshot_name = request->snapshot();
-        if (!mp::utils::valid_hostname(snapshot_name))
+        if (!snapshot_name.empty() && !mp::utils::valid_hostname(snapshot_name))
```

Comment:
Hey @sharder996, could you please fix the typos on the commit message? ("Don't throw...")